.github/workflows/deploy-spaces.yml

```diff
@@ -112,9 +113,7 @@ jobs:
         --gradio-version ${{ steps.get_gradio_version.outputs.GRADIO_VERSION }} > url.txt
         echo "SPACE_URL=$(cat url.txt)" >> $GITHUB_OUTPUT
     - name: Upload Website Demos
-      if: >
-        github.event == 'workflow_dispatch' &&
-        github.event.workflow_run.conclusion == 'success'
+      if: github.event_name == 'workflow_dispatch'
```

Comment:
we only have one workflow for this now, we so we don't need to check if it fails/succeeds.